spec/factories/subscriptions.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :subscription do
+    association :supplier
+    callback_url { 'http://foo.bar/?bla=bla' }
+    topic { '*' }
```

Comment:
slightly confused by this, as subscription doesn't appear to have a 'topic' property